ts-defs/index.d.ts

```diff
@@ -1543,6 +1543,201 @@ interface Assertion {
 
 }
 
+interface SslOptions {
+    key: string | Buffer;
```

Comment:
`SslOptions` is a combination of [SecureContextOptions](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options) and [TlsOptions](https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener). 
We will use type definition from the `@types/node` module. 
So, the final `SslOptions` definition will look like this:
```js
interface SslOptions {
        pfx?: string | Buffer | Array<string | Buffer | Object>;
        key?: string | Buffer | Array<Buffer | Object>;
        passphrase?: string;
        cert?: string | Buffer | Array<string | Buffer>;
        ca?: string | Buffer | Array<string | Buffer>;
        ciphers?: string;
        honorCipherOrder?: boolean;
        ecdhCurve?: string;
        clientCertEngine?: string;
        crl?: string | Buffer | Array<string | Buffer>;
        dhparam?: string | Buffer;
        secureOptions?: number; // Value is a numeric bitmask of the `SSL_OP_*` options
        secureProtocol?: string; // SSL Method, e.g. SSLv23_method
        sessionIdContext?: string;
        handshakeTimeout?: number;
        requestCert?: boolean;
        rejectUnauthorized?: boolean;
        NPNProtocols?: string[] | Buffer[] | Uint8Array[] | Buffer | Uint8Array;
        ALPNProtocols?: string[] | Buffer[] | Uint8Array[] | Buffer | Uint8Array;
        SNICallback?: (servername: string, cb: (err: Error | null, ctx: SecureContext) => void) => void;
        sessionTimeout?: number;
        ticketKeys?: Buffer;
}
```

Reply:
I suggest importing and using node typings directly

```ts
import { TlsOptions } from 'tls';
```

Reply:
We cannot use direct import because `Testcafe` doesn't ship with `@types/node`. This is why I suggest using a plain list of options.

Reply:
`@types/node` could be added as a dependency then with 
```ts
/// <reference types="node" />
```

Reply:
OK, after discussing this with @AndreyBelym, we decided to add the `@types/node` module as a dependency. 
Please add the corresponding changes to your pull request.

Reply:
Added a dependency